_posts/2024-04-02-shared-conda-tutorial.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: The ideal setup for a multi-user conda installation
+tags: conda bash systems-administration
+contributors:
+- boulund
+- pmenzel
+- hexylena
+---
+
+[Conda](https://docs.conda.io/projects/conda/en/stable/) and [Bioconda](https://bioconda.github.io/) have become crucial tools in most bioinformatician's toolkit. It is often used to install applications and create reproducible environments on their system or a compute cluster. 
+
+On multi-user systems (e.g. shared Linux servers), multiple users can share  the same conda installation and the environments thererin. This is a good idea and usually works well when only already existing environments are re-used, but can become tricky when users also want to create environments (using `conda create`) into the shared conda installation. On a standard Linux system, this often results in permission problems when (re-)downloading packages and overwriting files that were created by others.
+
+Here we describe a setup with a shared (read-only) conda installation, with re-usable shared environments, and private per-user environments.
+
+
+## Tutorial
+
+### Creating the shared conda installation
+
```

Comment:
Does the conda distribution matter? I.e. is this valid whether you are doing anaconda, miniconda or something else?

Reply:
I'm not sure about other variants than Anaconda and Miniconda, but it should work with [Miniforge](https://github.com/conda-forge/miniforge) as well since they all have the same CLI.